exec/java-exec/src/main/java/org/apache/drill/exec/ExecConstants.java

```diff
@@ -558,7 +558,7 @@ private ExecConstants() {
 
   /**
    * Timeout for create prepare statement request. If the request exceeds this timeout, then request is timed out.
-   * Default value is 10mins.
+   * Default value is 30 seconds.
```

Comment:
Is it 30 seconds or 10 seconds? In any case, I don't think that the entire comment provides any additional info that cannot be inferred from the code itself.

Reply:
Agree, with Vlad, it better to remove this sentence at all: `Default value is 30 seconds.`

Reply:
I refer to the entire comment. "If the request exceeds this timeout, then request is timed out" does not add any value, it is simply a definition for a timeout. The same for `CREATE_PREPARE_STATEMENT_TIMEOUT_MILLIS` and "Timeout for create prepare statement request".